sdk/cosmos/azure-cosmos-kafka-connect/src/docker/resources/sink.example.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "cosmos-kafka-connectv2",
+  "config": {
+    "connector.class": "com.azure.cosmos.kafka.connect.CosmosSinkConnector",
+    "tasks.max": "5",
+	"topics": "GreenTaxiRecords",
+    "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+    "value.converter.schemas.enable": "false",
+    "key.converter": "org.apache.kafka.connect.json.JsonConverter",
+    "key.converter.schemas.enable": "false",
+	"azure.cosmos.account.endpoint":"",
```

Comment:
nit might be better to fill these out with the `{endpoint}` and so on to be consistent with the other example

Reply:
updated